src/models/Room.ts

```diff
@@ -0,0 +1,19 @@
+import { Schema, model, Types } from 'mongoose';
+
+interface Room {
+    name: string;
+    sections: Types.Array<string>;
+    schedule: Types.Array<Record<string, string[]>>;
+    closed: boolean;
+}
+
+const roomSchema = new Schema<Room>({
+    name: String,
+    sections: [],
```

Comment:
Would a many to one relationship between sections and rooms make more sense?

Reply:
This will be added once the section module is finished